Makefile

```diff
@@ -38,7 +38,7 @@ release=--release
 endif
 
 .PHONY: setup
-setup: setup-qemu
+setup:
```

Comment:
I assume this change was accidental -- `setup` should depend on `setup-qemu`.

(Yes, I realize that QEMU is not used by `make test` at the moment, but it will be once we have integration testing in place again -- which depends on this console API.)

Reply:
Oops, yes. I keep this as a local change because it currently breaks for a reason I didn't bother to investigate. It also pulls in the entire tock-os, which I consider overkill.